<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pin</class>
 <widget class="QDialog" name="Pin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BankSimul</string>
  </property>
  <widget class="QLineEdit" name="InsertPinLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="InputPinLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Anna Pin:</string>
   </property>
  </widget>
  <widget class="QLabel" name="PinAttemptsLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Yrityksi√§:3</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="ProgressBar">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>20</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QPushButton" name="EnterPinButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
